src/base/LruMap.h

```diff
@@ -43,7 +56,7 @@ template <class EntryValue, size_t EntryCost = sizeof(EntryValue)> class LruMap
     /// Search for an entry, and return a pointer
     EntryValue *get(const char *key);
     /// Add an entry to the map
-    bool add(const char *key, EntryValue *t);
+    bool add(const char *key, EntryValue *t, int ttl);
```

Comment:
Nothing fake about the default TTL values, they are simply based on the latest squid.conf settings for the callers that do not (yet) use a more specific per-entry TTL. I looked at setting an implicit default value for the ttl parameter, however all of the existing callers use some form of actual TTL in various circumstances (albeit one stored in the LruMap instance), so the generic add() would not be used even if it existed.